src/test/java/api/loans/InHouseUseCheckInTest.java

```diff
@@ -0,0 +1,169 @@
+package api.loans;
+
+import static api.support.http.CqlQuery.queryFromTemplate;
+import static api.support.http.Limit.noLimit;
+import static api.support.http.Offset.noOffset;
+import static org.hamcrest.core.Is.is;
+import static org.hamcrest.junit.MatcherAssert.assertThat;
+
+import java.util.UUID;
+
+import org.folio.circulation.support.http.client.IndividualResource;
+import org.junit.Test;
+
+import api.support.APITests;
+import api.support.CheckInByBarcodeResponse;
+import api.support.MultipleJsonRecords;
+import api.support.builders.RequestBuilder;
+
+public class InHouseUseCheckInTest extends APITests {
```

Comment:
Now that there are unit tests for this behaviour, maybe the number of tests here can be reduced.

For example
* A test for when item is available, is not requested and is checked in to primary service point (positive flow)
* A test for when item item is requested (negative flow dependent upon loaded records)
* A test for when item item is checked in to other service point (negative flow dependent upon loaded records)

What do you think?

Reply:
removed. Also left a test when there are closed requests for item.